app/models/manageiq/providers/openstack/infra_manager/host.rb

```diff
@@ -412,11 +412,119 @@ def provide(taskid = nil)
       begin
         MiqEvent.raise_evm_job_event(self, :type => "provide", :suffix => "complete")
       rescue => err
-        $log.warn("Error raising complete provide event for #{log_target}: #{err.message}")
+        _log.warn("Error raising complete provide event for #{log_target}: #{err.message}")
       end
     end
 
     task.update_status("Finished", task_status, "Provide Complete with #{workflow_state}") if task
-    $log.info("Provide #{log_target}...Complete - Timings: #{t.inspect}")
+    _log.info("Provide #{log_target}...Complete - Timings: #{t.inspect}")
+  end
+
+  def supports_start?
```

Comment:
I'm not sure about `host_start`, because `Host` is already giving context.  So start is good for a feature. 
But please dont put the method in here like this. Rather use the `validate_start` like below, which is also defined in the base class.

We want to migrate every feature completely in a single PR